services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/EvaluationManagerImpl.java

```diff
@@ -442,22 +428,22 @@ void validateEvaluationRoundLimits(List<EvaluationRoundLimit> evaluationRoundLim
 		}
 	}
 
-	// TEMPORARY ADMIN CALL TO MIGRATE AL USAGES OF SubmissionQuotas INTO EvaluationRounds
 	@Override
-	public void adminMigrateSubmissionQuota(UserInfo userInfo){
-		if (!userInfo.isAdmin()){
-			throw new UnauthorizedException();
-		}
+	@WriteTransaction
+	public void migrateSubmissionQuota(UserInfo userInfo, String evaluationId){
+		Evaluation evaluation = validateEvaluationAccess(userInfo, evaluationId,ACCESS_TYPE.UPDATE);
 
-		List<Evaluation> evaluations = evaluationDAO.getAllEvaluations();
-		for(Evaluation evaluation: evaluations){
-			List<EvaluationRound> rounds = EvaluationRoundTranslationUtil.fromSubmissionQuota(evaluation, idGenerator);
+		ValidateArgument.requirement(evaluation.getQuota() != null, "The evaluation does not have an SubmissionQuota to convert");
```

Comment:
The evaluation does not have an SubmissionQuota to convert -> The evaluation does not have a SubmissionQuota to convert